spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -252,6 +257,8 @@ default void cleanup() {
 
 A repository may have at most one resource accessor method.
 
+A repository which declares a resource accessor method with return type `jakarta.persistence.EntityManager` is always a <<Stateful repositories,stateful repository>>.
```

Comment:
Here's an idea for turning this into a recommendation so that we maintain compatibility with Data 1.0,
```suggestion
A resource accessor method with return type `jakarta.persistence.EntityManager` should only be defined on a <<Stateful repositories,stateful repository>>.
```
Later, when we have entity agent, we could also write,
```
A resource accessor method with return type `jakarta.persistence.EntityAgent` must only be defined on a stateless repository.
```